testdata/configtest.json

```diff
@@ -2408,6 +2418,44 @@
      "supportedCurrencies": ""
     }
    ]
+  },
+  {
+   "name": "Ftx",
```

Comment:
Please make this in line with the one in config_example (currently it has null pairs)